velox/functions/prestosql/aggregates/RegisterAggregateFunction.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+namespace facebook::velox::aggregate {
+extern void registerApproxDistinct();
+extern void registerApproxMostFrequent();
+extern void registerApproxPercentile();
+extern void registerArbitraryAggregate();
+extern void registerArrayAggregate();
+extern void registerAverageAggregate();
+extern void registerBitwiseAggregate();
+extern void registerBoolAggregate();
```

Comment:
You can move all the extern void statements to the .cpp file and keep this header clean with just the registerAllAggFunctions declaration.

Reply:
All the extern void statements are moved to the .cpp file.